src/bsoncxx/test/CMakeLists.txt

```
@@ -12,24 +12,28 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-include_directories(
-    ${MONGO_CXX_DRIVER_SOURCE_DIR}/src/third_party/catch/include
-)
+if (${ENABLE_TESTS})
```

```suggestion
if (ENABLE_TESTS)
```

Fun `if` parsing oddities again.

---

I'm unresolving this because after applying this suggestion, the bsoncxx tests are being built by default. Should I revert this suggestion?

```
ls -l src/bsoncxx/test/test_bson
```